api/api/models/mission.py

```diff
@@ -0,0 +1,24 @@
+"""This module manages generic models required for missions
+
+The following models are present here:
+
+    - Auth: Mission
+"""
+from django.db import models
+
+from api.models import Auth, MAX_TITLE_LENGTH
+
+
+# TODO: add a Team one model will be available
+# Create role Manager ??
+class Mission(models.Model):
+    # Duration is in number of days.
+    duration = models.FloatField(default=1)
+    start = models.DateField(auto_now_add=True)
+    end = models.DateField(auto_now_add=True)
```

Comment:
Shouldn't these value be set by the user, instead of having `auto_now_add` set to True ?

The `REQUIRED_FIELDS` array is also missing, please add it.

Reply:
Done